Documentation/community/roadmap.rst

```diff
@@ -0,0 +1,218 @@
+security or urgent fixes. 
+
+Focus areas
+-----------
+
+For a finer-granularity view, and insight into detailed enhancements and fixes,
+please refer to `issues on GitHub <GitHub issues_>`_. 
+
+Welcoming New Contributors
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As a CNCF project we want to make it easier for new contributors to get involved
+with Cilium. This includes both code and non-code contributions such as
+documentation, blog posts, example configurations, presentations, training
+courses, testing and more. Check the :ref:`dev_guide` documentation to understand how to get
+involved with code contributions, and the `Get Involved`_ guide for guidance on
+contributing blog posts, training and other resources. 
+
+CNCF Graduation
+~~~~~~~~~~~~~~~
+
+Given the widespread adoption of Cilium, `CNCF Graduation`_ is a natural goal for
+the project. We believe that the majority of the criteria are already met, but
+the following still need to be addressed: 
+
+* `CII Best Practice Badge`_
+* Third-party security audit
+
+.. _rm-cilium-service-mesh:
+
+Cilium Service Mesh 
+~~~~~~~~~~~~~~~~~~~
+
+Our eBPF-accelerated Service Mesh is the main focus for
+major enhancement, and it's a natural evolution of Cilium's networking
+capabilities. We released a beta at the end of 2021 and had very valuable
+feedback from our user community. The next steps we'd like to take for Cilium
+Service Mesh (in no particular order) are: 
+
+* Integration of service mesh branch into mainline Cilium
+* Graduating Ingress to stable
+  
+  * 100% upstream conformant
+  * Visibility via Hubble
+  * Support for additional annotations
+ 
+* Graduating Prometheus metrics and OpenTelemetry collector to stable
+* Using Kubernetes as servicemesh control plane 
+ 
+  * Simple to use sidecar-free service mesh configured using Kubernetes Services
+    and Ingress with support for additional annotations
+
+* Graduating EnvoyConfig CRD to stable
+* Extended sample Grafana dashboards for L7 visibility
+* SMI integration 
+* SPIFFE integration
+* Gateway API Integration
+* Next-generation mutual authentication datapath framework
+
+  * Support for integrated runtime identity
+  * SSL-based mutual authentication
+  * Support for any network protocol
+
+* Performance benchmarking
+
+.. _rm-clustermesh:
+
+ClusterMesh
+~~~~~~~~~~~
+
+Core :ref:`ClusterMesh<clustermesh>` is stable and widely adopted. Future extensions include: 
+
+* Service affinity
+* Cluster health checks
+* :ref:`External Workloads<external_workloads>` graduating to stable
+
+
+.. _rm-advanced-networking:
+
+Advanced Networking features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several advanced networking features currently in beta, several of
+these are already in production use with a set of adopters. We expect the
+following features to graduate to stable:
+
+.. 
+   Docs comment: Wondering why the Maglev link looks different from the rest? For some 
```

Comment:
I'll have a look at it :)

Reply:
See my comment below https://github.com/cilium/cilium/pull/19540#issuecomment-1106651401